modules/core/src/matrix_transform.cpp

```diff
@@ -4,6 +4,7 @@
 
 #include "precomp.hpp"
 #include "opencl_kernels_core.hpp"
+#include <thread>
```

Comment:
Why do we need this?

Reply:
agreed, why is this needed?
the opencv wrappers around threading libraries should be used instead. STL thread is not universally used. For example, I use Intel TBB on my private builds of opencv.

Reply:
removed, sorry for the typo.
It is a test code to get thread id to insure parallel really works.